libs/common/src/types/tfm/handle-sso-redirect-form.ts

```diff
@@ -0,0 +1,26 @@
+import { AuthorizationUrlRequest } from '@azure/msal-node';
+import { Response } from 'express';
+import { EntraIdAuthCodeRedirectResponseBody } from './entra-id';
+import { TfmSessionUser } from './tfm-session-user';
+import { AuditDetails } from '../audit-details';
+import { CustomExpressRequest } from '../express-custom-request';
+import { ApiErrorResponseBody } from '../api-error-response-body';
+
+export type HandleSsoRedirectFormRequest = {
+  authCodeResponse: EntraIdAuthCodeRedirectResponseBody;
+  originalAuthCodeUrlRequest: AuthorizationUrlRequest;
+  auditDetails: AuditDetails<'system'>;
+};
+
+export type HandleSsoRedirectFormResponse = {
+  user: TfmSessionUser;
+  token: string;
+  expires: string;
+  successRedirect?: string;
+};
+
```

Comment:
Thoughts on `HandleSsoRedirectRequest` `HandleSsoRedirectResponse` etc instead of saying "form"?